apps/gro/fields/email-address.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  'email-text': {
+    mixin: 'input-text',
```

Comment:
no - strangely there isn't an email mixin. Guess we could make one but out of scope for this